lib/graphql/generators/install_generator.rb

```diff
@@ -0,0 +1,44 @@
+require 'graphql/generators/base_generator'
+module GraphQL
+  module Generators
+    # Add GraphQL to a Rails app with
+    # `rails g graphql:install`.
+    #
+    # Setup a folder structure for GraphQL:
+    #
+    # ```
+    # - app/
+    #   - graphql/
+    #     - resolvers/
+    #     - types/
+    #       - query_type.rb
+    #     - loaders/
+    #     - mutations/
+    #     - {app_name}_schema.rb
```

Comment:
Is there a reason for prefixing with `{app_name}_` besides the fact that otherwise we'd be conflicting with `GraphQL::Schema`?